### ISSUE TITLE
Hallux weakness

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11182]
**Preferred term label:**
Hallux dorsiflexion weakness
Hallux plantarflexion weakness

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For dorsiflexion:
 Hallux rigidus
Sesamoid injuries of the hallux

For dorsiflexion:
Sesamoid injuries of the hallux

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001374
- label: Hallux dorsiflexion weakness
- parents: [Limb muscle weakness (HP:0003690)](https://hpo.jax.org/app/browse/term/HP:0003690)
- pmid: 
- def: Reduced ability to lift the big toe (hallux) upward toward the shin.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11182
- synonyms:

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001375
- label: Hallux plantarflexion weakness
- parents: [Limb muscle weakness (HP:0003690)](https://hpo.jax.org/app/browse/term/HP:0003690)
- pmid: 
- def: Reduced ability to push the big toe (hallux) downward, away from the shin.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11182
- synonyms:
